src/server/routes/conversions.js

```diff
@@ -121,7 +123,7 @@ router.post('/addConversion', async (req, res) => {
 	const validationResult = validate(req.body, validConversion);
 	if (!validationResult.valid) {
 		log.error(`Invalid input for ConversionsAPI. ${validationResult.error}`);
-		res.sendStatus(400);
+        failure(res, 400, `Invalid input: ${validationResult.error}`);
```

Comment:
I'd probably do `"Invalid input: " + validationResult.error.toString()` just to stay consistent with the other failure calls, but I'll leave that up to the official reviewer